sdk/state/state.go

```diff
@@ -183,10 +193,15 @@ func (c *Channel) setInitiatorChannelAccountSequence(seqNum int64) {
 	c.initiatorChannelAccount().SequenceNumber = seqNum
 }
 
+// IsInitiator returns true if this channel initiated the process of opening a
+// channel, else false.
 func (c *Channel) IsInitiator() bool {
 	return c.initiator
 }
 
+// NextIterationNumber returns the next iteration number for the channel. If
+// there is a pending unauthoried close agreement, then that agreement iteration
+// is used, else the latest authorized agreeement is used.
 func (c *Channel) NextIterationNumber() int64 {
```

Comment:
💭 Maybe we should make this function not-exported?

Reply:
I'm not suggesting we address this in this PR, I'm just noticing this.

Reply:
🤔  trying to think if there's a use case for an external app needing to know. I guess only the channel needs to know it. Makes sense to me to not-export

though we'd have to change modify the integration tests that is using it for logging

Reply:
I don't think there's a need. It'd keep the API slimmer which would be nice. I'll have a look if the integration tests really need it.

Reply:
This doesn't need to block this Pr though, this PR is great.

Reply:
#437